contrib/scrooge/src/python/pants/contrib/scrooge/tasks/scrooge_gen.py

```diff
@@ -135,20 +141,25 @@ def execute_codegen(self, target, target_workdir):
     self._must_have_sources(target)
 
     namespace_map = self._thrift_defaults.namespace_map(target)
+    compiler_args = self._thrift_defaults.compiler_args(target)
 
     partial_cmd = self.PartialCmd(
         language=self._validate_language(target),
         rpc_style=self._validate_rpc_style(target),
```

Comment:
If the `rpc_style` arg is deprecated, it might be a good idea to remove it from `PartialCmd` now, and convert/merge it to the `compiler_args` list before creating the `PartialCmd`.